styles.css

```diff
@@ -0,0 +1,1128 @@
+  height: 80px;
+  font-size: 15px;
+  color: #fff;
+  border: none;
+  background-color: #ec5242;
+}
+
+.see-whole-project-container {
+  display: none;
+}
+
+.tutors-main-container {
+  width: 100%;
+  margin-top: 20px;
+  padding-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.featured-tutors-container {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.featured-tutors-text-container {
+  width: 375px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.featured-tutors-text {
+  font-size: 23px;
+  line-height: 32px;
+  font-weight: 500;
+}
+
+.short-line-1-container {
+  width: 50px;
+  height: 20px;
+}
+
+.short-line-1 {
+  height: 2px;
+  color: #ec5242;
+  margin-top: 2px;
+}
+
+.tutors-profile-container {
+  width: 100%;
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+.more-tutors-profile-container {
+  display: none;
+}
+
+.display-more-tutors {
+  width: 100%;
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+.tutor-profile {
+  width: 95%;
+  height: 150px;
+  display: grid;
+  grid-template-columns: 150px 65%;
+}
+
+.hidded {
+  display: none;
+}
+
+.tutor-profile-img-container {
+  height: 150px;
+}
+
+.tutor-profile-img {
+  width: 145px;
+  height: 150px;
+  float: right;
+}
+
+.tutor-details-container {
+  height: 150px;
+  display: flex;
+  flex-direction: column;
+}
+
+.tutor-name-container {
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+.tutor-name {
+  font-size: 20px;
+  font-weight: 600;
+  margin-left: 10px;
+  color: #272a31;
+}
+
+.tutor-educations-container {
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.tutor-education {
+  font-size: 15px;
+  font-weight: 500;
+  color: #ec5242;
+  margin-left: 10px;
+}
+
+.tutor-experience-container {
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.tutor-experience {
+  font-size: 13px;
+  font-weight: 500;
+  margin-left: 10px;
+}
+
+.see-more-button-container {
+  width: 95%;
+  height: 40px;
+  margin-top: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  column-gap: 10px;
+  border: 1px solid grey;
+}
+
+.hide-arrow-container {
+  display: none;
+}
+
+.down-arrow-container {
+  display: block;
+}
+
+.hide-more-tutors-container {
+  display: none;
+}
+
+.partner-text {
+  display: none;
+}
+
+.short-line-2 {
+  display: none;
+}
+
+.img-size {
+  display: none;
+}
+
+.footer-container {
+  display: none;
+}
+
+/* styles for about page */
+.home-page-title-container {
+  width: 100%;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.home-page-primary-text {
+  font-size: 25px;
+  font-weight: 400;
+  color: #ec5242;
+}
+
+.home-page-title-text {
+  width: 438px;
```

Comment:
- [ Optional ] Please remove this `width: 438px;` and add a `margin-top: 40px;` to the `.home-page-supporting-text-model` class.